scripts/db_migrator.py

```diff
@@ -53,11 +55,21 @@ def __init__(self, namespace, socket=None):
         self.TABLE_KEY       = 'DATABASE'
         self.TABLE_FIELD     = 'VERSION'
 
+        # when there is golden_config_db.json
+        # load config data from golden_config_db.json
+        # when there is no golden_config_db.json
         # load config data from minigraph to get the default/hardcoded values from minigraph.py
         # this is to avoid duplicating the hardcoded these values in db_migrator
         self.minigraph_data = None
         try:
-            if os.path.isfile(MINIGRAPH_FILE):
+            if os.path.isfile(GOLDEN_CFG_FILE):
+                with open(GOLDEN_CFG_FILE) as f:
+                    golden_data = json.load(f)
+                    if namespace is None:
+                        self.minigraph_data = golden_data
```

Comment:
>minigraph_data [](http://example.com/codeflow?start=29&length=14)

It's confusing to change the meaning of "minigraph_data". Suggest make it immutable, and create new var for new purpose.

If both GOLDEN_CFG_FILE and MINIGRAPH_FILE exists, the table inside GOLDEN_CFG_FILE will override table from MINIGRAPH_FILE. Please check with @wen587 and try reuse this override logic.